packaging/conda-recipe/viral-ngs-template/meta.yaml

```diff
@@ -0,0 +1,93 @@
+package:
+  name: viral-ngs
+  version: {{PKG_VERSION_CONDA}}
+
+about:
+  home: https://github.com/broadinstitute/viral-ngs
+  license: https://raw.githubusercontent.com/broadinstitute/viral-ngs/master/LICENSE
+  summary: A set of scripts and tools for the analysis of viral NGS data
+
+source:
+  fn: viral-ngs-{{PKG_VERSION}}.zip
+  url: https://github.com/broadinstitute/viral-ngs/archive/{{PKG_VERSION}}.zip
+
+build:
+    number: 0
+    rpaths:
+        - lib/
+        - lib//
+
+requirements:
+  build:
+    - python
+    - java-jdk >=7
+    - perl
+    {% for item in build_requirements %}
+    {{ item }}
+    {%- endfor %}
+    # C lib or compilation-related
+    - gcc   # [not osx]
+    - llvm  # [osx]
+    - boost 1.61* # needed for diamond, among other packages
+
+  run:
+    - python
+    - java-jdk >=7
+    - perl
+    {% for item in run_requirements %}
+    {{ item }}
+    {%- endfor %}
+    {% for item in py2_run_requirements %}
+    {{ item }} # [py2k]
+    {%- endfor %}
+    {% for item in py3_run_requirements %}
+    {{ item }} # [py3k]
+    {%- endfor %}
+    {% for item in linux_run_requirements %}
+    {{ item }} # [linux]
+    {%- endfor %}
+    {% for item in osx_run_requirements %}
+    {{ item }} # [osx]
+    {%- endfor %}
+    # C lib-related
+    - libgcc # [not osx]
+    - boost 1.61* # needed for diamond, among other packages
+
+test:
+  requires:
+    # testing-related
+    {% for item in test_requirements %}
+    {{ item }}
+    {%- endfor %}
+  commands:
+    # test command-line tools
+    - "assembly.py -h &> /dev/null"
+    - "broad_utils.py -h &> /dev/null"
+    - "illumina.py -h &> /dev/null"
+    - "interhost.py -h &> /dev/null"
+    - "intrahost.py -h &> /dev/null"
+    - "metagenomics.py -h &> /dev/null"
+    - "ncbi.py -h &> /dev/null"
+    - "read_utils.py -h &> /dev/null"
+    - "reports.py -h &> /dev/null"
+    - "taxon_filter.py -h &> /dev/null"
+
+    # test tool install
+    #- "cd $PREFIX/opt/$PKG_NAME-$PKG_VERSION && py.test -v test/unit/test_tools.py"
+
+    # various unit tests
+    #cd 
+    - "cd $PREFIX/opt/$PKG_NAME-$PKG_VERSION && py.test -v test/unit/test_tools_vphaser2.py"
```

Comment:
is it better to run these individually instead of lumping them all together in parallel (e.g. travis/tests-*.sh)?